Core/Aspects/Caching/CacheAspect.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Castle.DynamicProxy;
+using WebNetSample.Core.CrossCuttingConcerns.Caching;
+using WebNetSample.Core.Utilities.Interceptors;
+using WebNetSample.Core.Utilities.IoC;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace WebNetSample.Core.Aspects.Caching;
+
+public class CacheAspect : MethodInterception
+{
+    private readonly ICacheService _cacheService;
```

Comment:
I advise you to align class variables in the following order:
1. Constants
2. Private fields
3. Properties

Each of this group should be separated by empty row